packages/core/src/components/document-component/document-component.tsx

```diff
@@ -0,0 +1,246 @@
+   */
+  @Prop({ reflect: true })
+  open = false;
+
+  /**
+   * Marks this Document Component as belonging to an active filter.
+   */
+  @Prop({ reflect: true })
+  filtered = false;
+
+  /**
+   * Marks this Document Component as not-applicable.
+   */
+  @Prop({ reflect: true })
+  notApplicable = false;
+
+  /**
+   * When a child Document Component has a status "Draft".
+   */
+  @Prop({ reflect: true })
+  nestedDraft = false;
+
+  /**
+   * Marks as draft.
+   */
+  @Prop({ reflect: true })
+  draft = false;
+
+  /**
+   * Enables annotations.
+   */
+  @Prop({ reflect: true })
+  annotated = false;
+
+  /**
+   * Marks Document Component as reserved.
+   */
+  @Prop()
+  reserved = false;
+
+  /**
+   * Marks the Document Component as expired.
+   */
+  @Prop()
+  expired = false;
+
+  /**
+   * Type of Document Component.
+   */
+  @Prop({ reflect: true })
+  type?: string;
+
+  /**
+   * The wijzigactie as in STOP.
+   */
+  @Prop()
+  wijzigactie?: DocumentComponentWijzigactie;
+
+  /**
+   * Emitted when the user activates the toggle.
+   */
+  @Event({ bubbles: false })
+  dsoOpenToggle!: EventEmitter<DocumentComponentOpenToggleEvent>;
+
+  /**
+   * Emitted when the user activates the annotation button.
+   */
+  @Event({ bubbles: false })
+  dsoAnnotationToggle!: EventEmitter<DocumentComponentToggleAnnotationEvent>;
+
+  private get wijzigactieLabel(): string | undefined {
+    return this.wijzigactie && wijzigActieLabels[this.wijzigactie];
+  }
+
+  private handleHeadingClick = (e: MouseEvent) => {
+    if (this.type !== "LID") {
+      this.dsoOpenToggle.emit({ originalEvent: e, open: !this.open });
+    }
+  };
+
+  private suffix(): string | undefined {
+    if (this.expired) {
+      return "vervallen";
+    }
+
+    if (this.reserved) {
+      return "gereserveerd";
+    }
+
+    return undefined;
+  }
+
+  render() {
+    const suffix = this.suffix();
+
+    return (
+      <Host wijzigactie={this.wijzigactie}>
+        <div class="heading-container">
+          {this.wijzigactie && <span class="wijzigactie-label">{this.wijzigactieLabel}:</span>}
+          <div class={clsx(["heading", { [`type-${this.type?.toLowerCase()}`]: !!this.type }])}>
+            <Heading heading={this.heading} class="heading-element" onClick={this.handleHeadingClick}>
+              {this.type !== "LID" && (
+                <button id="toggle" type="button">
+                  <dso-icon icon={this.open ? "chevron-down" : "chevron-right"}></dso-icon>
+                </button>
+              )}
+              <div class="title">
+                {this.notApplicable && <span class="not-applicable">Niet van toepassing:</span>}
+                <dso-ozon-content content={this.label} inline></dso-ozon-content>{" "}
+                <dso-ozon-content content={this.nummer} inline></dso-ozon-content>{" "}
+                <dso-ozon-content content={this.opschrift} inline></dso-ozon-content>
```

Comment:
Die `{" "}` komen door formatting denk ik?

Reply:
Nee, die staan er bewust. JSX plaatst geen whitespace tussen elementen, maar hier wil ik een spatie tussen label, nummer en opschrift. En dan is dit de enige optie 🤷